include/common/logger.h

```diff
@@ -1 +1,55 @@
-class logger{};
\ No newline at end of file
+#pragma once
+#include <string>
+#include <fstream>
+#include <ctime>
+#include <chrono> 
+
+class Logger {
+public:
+	enum loglevel
```

Comment:
don't concatenate words like this. instead prefere dash. e.g. "log_level" in utility classes